pycmr/granule.py

```diff
@@ -17,6 +18,12 @@ def __init__(self):
         self.params = {}
         self.options = {}
 
+    def _urlEncodeString(self, input):
```

Comment:
Now, unless there's some expected future code that might go in here, I'm gonna blow a babs mind real quick:

`from urllib.parse import quote as urlEncodeString`

In addition, I recommend url_encode_string for consistency with our other methods + PEP8.